string arduino_id
uint16 vcc
